src/fontstash.h

```diff
@@ -781,6 +822,12 @@ void fonsSetFont(FONScontext* stash, int font)
 	fons__getState(stash)->font = font;
 }
 
+void fonsSetOversample(FONScontext* stash, int oversampleH, int oversampleV)
+{
+	fons__getState(stash)->oversampleH = oversampleH;
```

Comment:
Inconsistent... would you like other fonsSet* functions to include validation? Or is there a good reason to only use here? **or** perhaps oversampling should simply be on/off (2x2 gives good results, after that can be questionable). Perhaps it should be automatic if pixel snapping is off? What do you think?